frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -88,6 +92,7 @@ function mapStateToProps(state: State) {
 function EmbedFrame({
   className,
   children,
+  header,
```

Comment:
nit: We use a bunch different things to make the header. Would prefer a different name here, so we know exactly what we are receiving in this.
```suggestion
  dashboardTabs,
```